packages/eslint-plugin/tests/rules/unbound-method.test.ts

```diff
@@ -291,6 +320,9 @@ function foo(arg: ContainsMethods | null) {
       'const { unbound } = instance.unbound;',
       'const { unboundStatic } = ContainsMethods.unboundStatic;',
```

Comment:
These tests are just wrong so we don't need them - you just corrected them below

```suggestion
```

Reply:
@bradzacher  👍  Sure! I removed it. Thanks for the review